src/components/ConfirmCard/index.tsx

```diff
@@ -75,17 +93,49 @@ export function ConfirmCard({
             justifyContent: 'flex-end',
             margin: '0.5em 0',
             gap: 1,
+            [theme.breakpoints.up('sm')]: {
+              flexDirection: 'column',
+            },
           }}
           className="button-container"
           display="flex"
         >
+          {includeCheckbox && (
+            <FormGroup>
+              <FormControlLabel
+                sx={{
+                  color: palette.white.main,
+                  '& .MuiFormControlLabel-label': {
+                    fontSize: '16px',
+                    fontFamily: 'Roboto',
+                    zIndex: 1,
+                    [theme.breakpoints.up('sm')]: {
+                      fontSize: '14px',
+                    },
+                  },
+                }}
+                label={t('ConfirmCard.checkbox.label')}
+                control={
+                  <Checkbox
+                    sx={{ color: palette.white.main }}
+                    checked={isNotifyChecked}
+                    onChange={(event) => {
+                      setIsNotifyChecked(event.target.checked);
+                      onNotifyChange && onNotifyChange(event.target.checked);
+                    }}
+                  />
+                }
+              />
+            </FormGroup>
+          )}
```

Comment:
On smaller screens this checkbox and its label move on the side of cancel and confirm buttons and is therefore partially invisible.

Reply:
Done :+1: